coresimd/x86/mod.rs

```diff
@@ -512,3 +512,13 @@ pub use self::rdrand::*;
 
 mod sha;
 pub use self::sha::*;
+
+#[cfg(test)]
+use stdsimd_test::assert_instr;
+
+/// Generates the trap instruction `UD2`
+#[cfg_attr(test, assert_instr(ud2))]
+#[inline]
+pub unsafe fn ud2() -> ! {
```

Comment:
The `stdsimd-verify` crate (https://github.com/rust-lang-nursery/stdsimd/tree/master/crates/stdsimd-verify) needs to be updated to support bottom.

Reply:
Here: https://github.com/rust-lang-nursery/stdsimd/blob/master/crates/stdsimd-verify/src/lib.rs#L127